src/pymedphys/dicom.py

```diff
@@ -0,0 +1,8 @@
+from pymedphys_dicom.dicom import (
```

Comment:
Nice to see removal of `import *`

Reply:
Yup, and we need to remove that also from all the levelling. With this new format each module at each level depth is going to be explicit about what API it is exposing. That will make versioning much clearer, allow us to keep a stable API, while still changing lots of things. ... them OOP folk and their encapsulation :). (only psudo encapsulation, because people can still dive in past the API, but we won't provide guarantees for that sort of thing).